src/calculateRentalCost.js

```diff
@@ -4,7 +4,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  let total = days * 40;
```

Comment:
I do not understand what means 40, 7, 50, 3, 20? create constants with understandable names for these numbers